src/pymovements/__init__.py

```diff
@@ -79,3 +81,7 @@
 ]
 
 __version__ = _version.get_versions()['version']
+
+LOG_LEVEL = os.environ.get('LOG_LEVEL', 'INFO')
+LOG_CONFIG = '[%(levelname)s] %(asctime)s %(name)s:%(lineno)d - %(message)s'
+logging.basicConfig(level=LOG_LEVEL, format=LOG_CONFIG)
```

Comment:
I needed a bit of time to think about this.

we probably shouldn't set this in the package, as a user might have a custom logging config defined. after importing pymovements their config would be overwritten.

this logic should actually be on the side of the user, so I would suggest to revert the changes in this file.

still, defining logging levels and other configuration aspects can form a small mini tutorial. this would be out of scope of this PR but feel free to create one in case you like.

Reply:
indeed, will revert
although I'm not sure it's overwriting the user logging config, this should be only at the package level; but I'll have to check; on the long run probably a `logging.ini` would be more appropriate